escheduler-ui/src/js/conf/home/pages/dag/_source/formModel/tasks/http.vue

```diff
@@ -0,0 +1,148 @@
+<template>
+  <div class="flink-model">
```

Comment:
Here, the flink-model should be modified to http-model, please also change the css part for this at the bottom of this file